dependencies.gradle

```diff
@@ -38,8 +38,12 @@ def VERSIONS = [
         'info.ganglia.gmetric4j:gmetric4j:latest.release',
         'io.prometheus:simpleclient_common:latest.release',
         'io.prometheus:simpleclient_pushgateway:latest.release',
+        // TODO support HttpTagsProvider variants based on Jakarta APIs?
+//        'jakarta.servlet:jakarta.servlet-api:latest.release',
```

Comment:
In working on this, I noticed we have a javax servlet based http tags provider. We should probably also have jakarta-based one. Supporting either in the instrumentation may be a challenge. But something to work on separately from this PR.